ecogvis/signal_processing/processing_data.py

```diff
@@ -346,22 +346,23 @@ def high_gamma_estimation(block_path, bands_vals, new_file=''):
         lfp = nwb.processing['ecephys'].data_interfaces['LFP'].electrical_series['preprocessed']
         rate = lfp.rate
 
-        nBands = len(band_param_0)
+        nBands = 40
```

Comment:
Why are you hard-coding 40?

Reply:
Line 72 in `process_nwb.wavelet_transform` is `log_spaced_cfs(4.0749286538265, 200, 40)` which always generates 40 bands. So, the shapes of `X_analytic` and `X_fft_h` are always `(...,...,40)` and `(41,...)`